heat/core/linalg/basics.py

```diff
@@ -827,6 +827,225 @@ def norm(a):
     return exponential.sqrt(d).item()
 
 
+def outer(a, b, out=None, split=None):
+    """
+    Compute the outer product of two 1-D DNDarrays.
+
+    Given two vectors, ``a`` = [a_0, a_1, ..., a_n] and ``b`` = [b_0, b_1, ..., b_m], the outer product is:
```

Comment:
This is not essential, perhaps useful though, could we have the Mathjax version of the docstrings here? I really liked the way it displayed in the Github message.

Reply:
Thanks @Markus-Goetz ,

Now I have it down as 

```
.. math::

        
           a_0 \times b_0  & a_0 \times b_1 & . & . &  a_0 \times b_m \\
           a_1 \times b_0 & a_1 \times b_1 & . & . & a_1 \times b_m \\
           . & . & . & . & .   \\
           a_n \times b_0 & a_n \times b_1 & . & . & a_n \times b_m

```

but according to the sphynx documentation what we really want is 

```
.. math::
        :nowrap:

        \begin{pmatrix}
           a_0 \times b_0  & a_0 \times b_1 & . & . &  a_0 \times b_m \\
           a_1 \times b_0 & a_1 \times b_1 & . & . & a_1 \times b_m \\
           . & . & . & . & .   \\
           a_n \times b_0 & a_n \times b_1 & . & . & a_n \times b_m
        \end{pmatrix}
```

except flake8 complains about `invalid escape sequence '\e'`. Haven't found a good solution yet.

Reply:
OK Done now, thanks @Markus-Goetz and @Cdebus